google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UploadOfflineConversions.java

```diff
@@ -0,0 +1,162 @@
+    } catch (FileNotFoundException fnfe) {
+      System.err.printf(
+          "Failed to load GoogleAdsClient configuration from file. Exception: %s%n", fnfe);
+      return;
+    } catch (IOException ioe) {
+      System.err.printf("Failed to create GoogleAdsClient. Exception: %s%n", ioe);
+      return;
+    }
+
+    try {
+      new UploadOfflineConversions().runExample(
+          googleAdsClient,
+          params.customerId,
+          params.conversionActionId,
+          params.gclid,
+          params.conversionTime,
+          params.conversionValue);
+    } catch (GoogleAdsException gae) {
+      // GoogleAdsException is the base class for most exceptions thrown by an API request.
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
+      int i = 0;
+      for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()) {
+        System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+      }
+    }
+  }
+
+  /**
+   * Runs the example.
+   *
+   * @param googleAdsClient the Google Ads API client.
+   * @param customerId the client customer ID.
+   * @param conversionActionId conversion action ID associated with this conversion.
+   * @param gclid the GCLID for the conversion.
+   * @param conversionTime  date and time of the conversion.
+   * @param conversionValue the value of the conversion.
+   */
+  private void runExample(
+      GoogleAdsClient googleAdsClient,
+      long customerId,
+      long conversionActionId,
+      String gclid,
+      String conversionTime,
+      Double conversionValue) {
+    // Gets the conversion action resource name.
+    String conversionActionResourceName =
+        ResourceNames.conversionAction(customerId, conversionActionId);
+
+    // Creates the click conversion.
+    ClickConversion clickConversion = ClickConversion.newBuilder()
+        .setConversionAction(StringValue.of(conversionActionResourceName))
+        .setConversionDateTime(StringValue.of(conversionTime))
+        .setConversionValue(DoubleValue.of(conversionValue))
+        .setCurrencyCode(StringValue.of("USD"))
+        .setGclid(StringValue.of(gclid))
+        .build();
+
+    // Creates the conversion upload service client.
+    try (ConversionUploadServiceClient conversionUploadServiceClient =
+        googleAdsClient.getLatestVersion().createConversionUploadServiceClient()) {
+      // Uploads the click conversion. Partial failure should always be set to true.
+      UploadClickConversionsResponse response = conversionUploadServiceClient
+          .uploadClickConversions(
+              Long.toString(customerId),
+              ImmutableList.of(clickConversion),
+              true,
```

Comment:
Please add comments here to indicate the parameters for the true and false values, .e.g,

```
   // Enable partial failure
   true,
   // Disable validate only
   false
```